src/main/scala/bank/Controller.scala

```diff
@@ -0,0 +1,43 @@
+package bank
+
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+import akka.http.scaladsl.model.StatusCodes._
+import akka.http.scaladsl.server.Directives._
+import spray.json._
+
+class Controller extends DefaultJsonProtocol with SprayJsonSupport {
+
+  var accounts = List[Account](
+    Account(
+      accountNumber = "123",
+      currentBalance = 9999,
+      currency = "PHP",
+      customerFirstName = "Lloyd",
+      customerLastName = "Edano"
+    )
+  )
+
+  def createAccount = post {
+    path("api" / "accounts" ) {
+      entity(as[Account]) { account=>
+        accounts = accounts :+ account
+        complete(Created, "Account created".toJson )
+      }
+    }
+  }
+
+  def getAccount = get {
+    path ("api" / "accounts" / Segment) { accountNumber=>
+      accounts.find(_.accountNumber == accountNumber) match {
+        case Some(accountFound)=>
+          complete(OK, s"Account found: ${accountFound.toJson}".toJson )
```

Comment:
response body is not client friendly
`"Account found: {\"accountNumber\":\"qerty456\",\"currency\":\"USD\",\"currentBalance\":555.55,\"customerFirstName\":\"JC\",\"customerLastName\":\"Claridad\"}"`

change to
`{
  "accountNumber":"qerty456", 
  "currency":"USD",
  "currentBalance":555.55,
  "customerFirstName":"JC",
  "customerLastName":"Claridad"
}`

Reply:
this suggested change is no longer valid since get account was moved to new PR